src/display/xfa_layer.js

```diff
@@ -161,13 +161,22 @@ class XfaLayer {
       }
     }
 
+    /*
+       TODO: re-enable that stuff once we've JS implementation.
```

Comment:
Can you mention https://bugzilla.mozilla.org/show_bug.cgi?id=1719465 in this comment?